week5/07-chietnyie-eliemienty/task.py

```diff
@@ -0,0 +1,12 @@
+def main():
+    input_list = input().split(" ")
+    res = []
+    for el in input_list:
+        el = int(el)
```

Comment:
mutable variable are complex to read and support, !especially! when you change variable nature (here it stores strings and ints). try to avoid variable reassignment (except may be case, when it was None before)